### PR TITLE
EWPP-3036: Resubmit rejected requests.

### PR DESCRIPTION
**Pull request by upchuk**

### CODE REVIEW

**Review by 22Alexandra — COMMENTED** [review]
Minor comments.

**Comment by hernani** [comment]
Not related with this PR, but just leaving it here:
- When selecting today as the translation request deadline it fails with the exception that request deadline cannot be in the past.

**Comment by upchuk** [comment]
@hernani Fixed here https://github.com/openeuropa/oe_translation/pull/199/files#diff-70f7728e812813aa1dd7cad8661b81c04bad23bf806bd97c21be518ba02140a1R277

File: modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php [hunk 1]
```diff
@@ -1270,4 +1271,143 @@ public function testProductNotifications(): void {
...
+    $request = $this->createNodeTranslationRequest($node, TranslationRequestRemoteInterface::STATUS_REQUEST_FINISHED, [
+      [
+        'langcode' => 'fr',
+        'status' => TranslationRequestEpoetryInterface::STATUS_LANGUAGE_ACTIVE,
+      ],
+    ]);
+    $request->setEpoetryRequestStatus(TranslationRequestEpoetryInterface::STATUS_REQUEST_REJECTED);
+    $request->save();
+
+    // Now create a new request for this node that is active. Note that because
```
Comment (hernani) [inline modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php:1291]: that's because

**Review by hernani — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

File: modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php (hunk 1 above)
Comment (upchuk) [reply modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php:1291]: Nope :D

**Pull request merged**